FirebaseStorage/Sources/FIRStorage_Private.h

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  */
 
+#import <FirebaseStorageInterop/FIRStorageInterop.h>
```

Comment:
I think in the case we're trying to solve here this needs to be in the public header, otherwise the type system won't be able to see if the Storage instance actually conforms to that protocol or not. I *could* be wrong here and please speak up if you think it can be private :)